helm/cert-manager-app/charts/giantswarm-cluster-issuer/templates/np.yaml

```diff
@@ -14,8 +14,8 @@ spec:
   podSelector:
     matchLabels:
       job-name: {{ .Values.name }}
-  ingress:
-  - {}
+  # deny ingress traffic
+  ingress: []
```

Comment:
there's no need to receive ingress traffic - it's just kubectl applying a couple of yamls